.github/workflows/test_openvino.yml

```diff
@@ -33,12 +34,9 @@ jobs:
         # install PyTorch CPU version to avoid installing CUDA packages on GitHub runner without GPU
         pip install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cpu
         pip install .[openvino,nncf,tests,diffusers]
+    - name: Install openvino-nightly (optional)
```

Comment:
It make sense to have tests with `openvino-nightly`, could make sense to add it in the nightly tests instead, wdyt ?

Reply:
@echarlaix I think that for now it would be useful to have it here, so we can test the PRs that target 2023.3. And then after 2023.3 we can move this to nightly tests?

Reply:
Yes the only issue is that the CI will be red until the next release (as the quantization tests will fail for one at least of the openvino version), will merge it anyway if that helps on your side!